<?xml version="1.0"?>

<launch>
  <arg name="output_type"    default="screen"/>
  <arg name="folder"         default="$(find evimo)/config"/>
  <arg name="fps"            default="40"/>

  <node pkg="evimo" name="evimo_vis" type="datagen_online" output="$(arg output_type)" respawn="false" required="true">
    <param name="folder"         value="$(arg folder)"/>
    <param name="fps"            value="$(arg fps)"/>
  </node>

  <!-- Start rviz visualization with preset config -->
  <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find evimo)/launch/evimo.rviz" required="true"/-->
</launch>
